src/main/java/htsjdk/samtools/fastq/FastqRecord.java

```diff
@@ -228,8 +228,13 @@ public String toFastQString() {
     }
 
     /**
-     * Returns {@link #toFastQString()}
+     * Returns {@link #toFastQString()}.
+     *
+     * @deprecated since 07/2017. This method will be deprecated until it is changed in the future
+     * for a simpler representation of the object. For code relying on the formatting as a FASTQ
+     * String, please refactor your code to use {@link #toFastQString()}.
      */
+    @Deprecated
```

Comment:
I like the idea of warning people, but in general but I'm not sure what the behavior of deprecating a method on Object is... It seems strange.